litellm/integrations/langfuse.py

```diff
@@ -160,9 +161,7 @@ def log_event(
             )
             verbose_logger.info(f"Langfuse Layer Logging - logging success")
         except:
-            traceback.print_exc()
-            print(f"Langfuse Layer Error - {traceback.format_exc()}")
-            pass
+            print_verbose(f"Langfuse Layer Error - {traceback.format_exc()}")
```

Comment:
don't you still need the `pass` ?

Reply:
`pass` should only be needed if there is no code at all in the `except` block, because there is the `print_verbose` it is not necessary (and it was necessary previously as well)